tests/behaviour/array_storage_index_zeroed_test_test.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import pytest
+from starkware.starknet.compiler.compile import compile_starknet_files
+from starkware.starknet.testing.state import StarknetState
+from yul.main import transpile_from_solidity
+from yul.starknet_utils import deploy_contract, invoke_method
+
+warp_root = os.path.abspath(os.path.join(__file__, "../../.."))
+test_dir = __file__
+
+
+@pytest.mark.asyncio
+async def test_address():
```

Comment:
Behaviour tests don't actually need cairo-files. We can get this cairo files by transpiling from solidity. Furthermore, it is useful as it allows to keep .sol and .cairo files out of sync. You can take a look at `for-loop-no-condition_test.py` to see how it can be done